.codespellignore

```diff
@@ -4,3 +4,4 @@ geting
 ot
 intepreted
 updat
+ShouldNot
```

Comment:
could you try changing this to

```suggestion
shouldnot
```

Seems that there is a known bug where it only wants lowercase https://github.com/codespell-project/codespell/issues/2451#issuecomment-1269314113

Reply:
that works!